src/envoy/http/jwt_auth/token_extractor.cc

```diff
@@ -84,9 +84,10 @@ void JwtTokenExtractor::Extract(
     if (entry) {
       std::string token;
       absl::string_view val = entry->value().getStringView();
-      if (val.rfind(" ") != absl::string_view::npos) {
```

Comment:
actually, since using `rfind` didn't simplify anything, let's use `find` instead. search from right is likely slower (prefix is short and token is long).

Reply:
updated, PTAL.